### DISCUSSION TITLE
SEI model options and their references

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3877] [Q&A]
I want to know the different equations used in the model options provided by pybamm for SEI, like SEI solvent diffusion limited uses which set of equations, and SEI porosity change uses which set of equations.

Likewise, I would like to know about the set of equations used for lithium plating options and LAM options.
Thanks in advance.

**Comment by valentinsulzer** [comment]
The easiest way to do this is to go to the API docs and search for e.g. "SEI" or "lithium plating" or "loss of active material". That will give you the name of the submodel, and references for papers. Then on github you can search for that submodel if you want to see the source code and specific equations.
